style.css

```diff
@@ -409,3 +411,358 @@ hr {
+    justify-content: center;
+    border-bottom-left-radius: 100px;
+    background: url("./headerbg.svg");
+    background-repeat: no-repeat;
+    background-position: 60% 20%;
+  }
+
+  h1.name {
+    width: 920px;
+    height: 122px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin-top: 359px;
+  }
+
+  p.description {
+    width: 920px;
+    height: 98px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    margin-top: 12px;
+  }
+
+  .connect {
+    width: 117px;
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin-left: 43%;
+    margin-right: 90%;
+    margin-top: 0%;
+  }
+
+  ul.links {
+    padding: 0;
+    gap: 16px;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: row;
+    align-items: flex-start;
+    width: 117px;
+    height: 24px;
+    margin-right: 1060px;
+    margin-left: 260px;
+    margin-top: 14px;
+  }
+
+  nav a {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    text-decoration: none;
+    width: 250px;
+    height: 20px;
+    color: #6070ff;
+    display: flex;
+    align-items: center;
+    padding-left: 6%;
+  }
+
+  /* Desktop Projects Area */
+  .projects {
+    display: none;
+  }
+
+  .desktop-projects {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 142px;
+    margin-left: 149px;
+  }
+
+  .desktop-projects .desktop-project {
+    display: flex;
+    flex-direction: row;
+    background-color: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    margin-bottom: 142px;
+    width: 100%;
+  }
+
+  .desktop-project-right {
+    display: flex;
+    flex-direction: row-reverse;
+    background-color: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    margin-bottom: 142px;
+    width: 100%;
+  }
+
+  .desktop-projects .desktop-project .proj-image {
+    padding: 24px 0 24px 24px;
+  }
+
+  .desktop-project-right .desktop-project .proj-image {
+    padding: 24px 22px 24px 0;
+  }
+
+  .desktop-project-right .proj-desc {
+    padding: 48px 0 0 48px;
+  }
+
+  .desktop-projects .desktop-project .proj-desc {
+    padding: 48px 46px 0 46px;
+  }
+
+  .desktop-project-right .proj-desc .para {
+    width: 496px;
+    height: 70px;
+  }
+
+  .desktop-projects .desktop-project .proj-desc .para {
+    width: 496px !important;
+    height: 50px !important;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .about {
+    display: none;
+  }
```

Comment:
- [ ] As I mentioned about implementing responsivity, having multiple markups for different screens and using `display: none` for some markups in screens that you dont wish render them is a wrong approach. Kindly implement one markup with different stylings for different screens.